homemate/src/components/passwordRecovery/Email/index.js

```diff
@@ -0,0 +1,63 @@
+import React, { useState } from 'react';
+import { toast } from 'react-toastify';
+import Input from '../../input';
+import BaseButton from '../../button';
+import { PASSWORD_RECOVERY_CODE } from '../../../utils/constants';
+import './Email.css';
+import api from '../../../api';
+
+const stylesInvalid = {
+  label: {
+    color: 'red',
+  },
+};
+
+const stylesValid = {
+  label: {
+    color: '#F4F4F4',
+  },
+};
+
+function Email(props) {
+  const { setState, email, setEmail } = props;
+  const [labelEmailStyle, setLabelEmailStyle] = useState({});
+
+  const validateEmail = () => {
+    const validation = email === '' || email === null;
+    setLabelEmailStyle(validation ? stylesInvalid : stylesValid);
+
+    return !validation;
+  };
+
+  const recovery = () => {
+    if (validateEmail()) {
+      const body = {
+        email,
+      };
+
+      api
+        .post('/user/forget/password', body)
+        .then(() => {
+          toast('Código enviado');
+          setState(PASSWORD_RECOVERY_CODE);
+        })
+        .catch((error) => {
+          let msg = '';
+          if (error.response) msg = error.response.data.error;
+          else msg = 'Network failed';
+          toast.error(msg);
+        });
+    }
+  };
```

Comment:
Quando coloco o email e clico em enviar email, demora uns segundos para mudar de tela. Coloca o componente de loading por favor.

Reply:
eu pensei em fazer isso mas nessa branch ainda nao tem ele, estava pensando em adicionar depois de fazer o merge com a  develop que tem tudo, que fiquei com medo de pegar cosia que ja ta lá e depois dar aquelas bugadas de a branch estar afrente e atras ao mesmo tempo

Reply:
É so tu atualizar a develop que tu tem local, depois fazer o merge com a tua branch, localmente. 
posso fazer com você depois, se preferir

Reply:
Você diz de fazer o merge meio que manual do componente de loading? que se for manual acho q consigo fazer sosinho agora ser for por comando do git ai eu eu tenho medo de zuar a branch :C